Update dev docs for consistency (#91)
material-components/material-components-flutter: docs/components/text_fields.md

### Types

> The text field class consists of the following types: 1\. [filled text](#filled-text) 2\. [outlined text](#outlined-text)

Reviewer: nit:
```suggestion
There are two types of text fields: 1\. [filled text](#filled-text) 2\. [outlined text](#outlined-text)
```

Author: Good idea, done
